core/src/com/geekbrains/td/Monster.java

```diff
@@ -7,23 +7,23 @@
 import com.badlogic.gdx.math.Vector2;
 
 public class Monster implements Poolable {
-    private GameScreen gameScreen;
+    protected GameScreen gameScreen;
```

Comment:
Try to avoid direct protected fields access and use protected getters and setters instead. Consider this as a recommendation until you get into a situation, when you understand why direct fields access is bad.